### ISSUE TITLE
Fields in `article_data.csv`

### ISSUE THREAD

**Issue by adamhsparks** [openplantpathology/Reproducibility_in_Plant_Pathology#13]
There are some fields in the `article_data.csv` file that are already described elsewhere or can be easily derived from the citation.

However, fields like "country" or "Open_or_Restricted" or "Reproducibility_Instructions" can be interpreted and filled in various ways.

I'd propose the following, tell me if I'm off here, @zachary-foster, since you set them up.

* **country** - Country of publisher, _e.g._ Springer Netherlands, entered as the 3 letter ISO code, NLD  
* **Open_or_Restricted** - entered as "Open"/"Restricted"
* **Reproducibility_Instructions** - entered as "yes"/"no"

**Comment by adamhsparks** [comment]
I've added a "Publisher" field as well.

**Comment by zachary-foster** [comment]
Yea, we should have a document somewhere that says how each field should be filled out in detail.
I like the 3 letter country code. For fields that have only two options, I suggest using TRUE/FALSE and rephrasing the field accordingly. For example "Open_or_Restricted" could be "is_open" or just "open". 

Some of the field names are a bit long. Would you mind @adamhsparks if I renamed them to be shorter when possible and all lower case so that they are easier to use in code during the analysis?

**Comment by adamhsparks** [comment]
Be my guest, I agree with that sentiment. Make them boolean and use lowercase throughout.

**Comment by zachary-foster** [comment]
Cool, will do

**Comment by adamhsparks** [comment]
Impact factor. Should we use most recent or five year? Most recent is probably the easiest to find, from what I'm seeing.

**Comment by zachary-foster** [comment]
Most recent should be fine if that is easier. IF at the time the article was published might be the best metric, but probably more effort than it is worth

**Comment by adamhsparks** [comment]
Well, now that I think about it, we're looking at the past five years, the five year impact factor would make the most sense.

Scopus goes back to 2011 right now, but it give CiteScore and not all journals are in the index, _e.g._ Revista Mexicana de Fitopatología. But most of them are I think.

**Comment by zachary-foster** [comment]
If we can get most of them, that should be fine. 5 year IF sounds good to me

**Comment by adamhsparks** [comment]
@zachary-foster, I was working with setting up the Rmd file to write to a Google Sheet so I shortened the names in that. I hope that's OK?

**Comment by zachary-foster** [comment]
Yea, that is fine by me.

**Comment by adamhsparks** [comment]
In the "page_charges" column, I'm assuming this is just TRUE/FALSE. Are we referring only to the article or the journal itself since this may vary by journal depending on open access or not, etc.?

**Comment by adamhsparks** [comment]
@zachary-foster I finally read your vignette on the fields, I am sorry. I keep getting pulled away from this to other things. I made a few comments and spelling corrections there.

I'm fixing a few more issues with the final evaluation sheet output from R, duplicated rows mainly, right now and hope to have it ready to go this afternoon.

**Comment by adamhsparks** [comment]
I think that I have the [article_notes](https://docs.google.com/spreadsheets/d/1pwUnvJgcnvwg-b8vLEbrdAbIpS6Rdbd0lIiZJi2-GfI/edit?usp=sharing) sheet ready for our inputs, finally. I've locked the other two now since we should edit the third, final table.

This latest version has the impact factor, thanks @grunwald, and the bibliography data that I was able to retrieve using the DOI.

The columns for the reproducibility scoring are on the far right of the sheet. To the left are bibliographic data and other information like my notes on why an article was selected, etc.

**Comment by zachary-foster** [comment]
Hi @adamhsparks. No worries, I have not looked at this in a while either. I was in the forest with no usable Internet for the past few days, so I am just seeing this now. The spread sheet looks good. If we are ok with the fields and their definitions, then we should each do a few articles and then have a conference call or something to decide if we want any last minute changes. I will start on mine. I see @grunwald already did a few.

**Comment by emdelponte** [comment]
Hello! I started to download the articles. I am planning to do a few and then we can talk.

**Comment by zachary-foster** [comment]
Hi @adamhsparks, which sheet in the [article_notes](https://docs.google.com/spreadsheets/d/1pwUnvJgcnvwg-b8vLEbrdAbIpS6Rdbd0lIiZJi2-GfI/edit?usp=sharing) are we entering data in? I noticed some rows filled out in the "template" sheet, but none in the other two. Also the first two sheets are locked for me. Thanks

**Comment by adamhsparks** [comment]
It's the one that is not locked. The foulest two are used in preparation of
the third.

-- 
Adam H Sparks

**Comment by adamhsparks** [comment]
Bugger, I just realised that the third sheet doesn't have all of the assignees! I don't know what happened.

**Comment by adamhsparks** [comment]
I've fixed that issue. I'll move any of the evaluations over to the new sheet and clean up my mess.

You all shouldn't see any difference other than all articles have been assigned but may not have full bibliographic information.

**Comment by adamhsparks** [comment]
All-
I've highlighted the fields that we need to pay attention to and enter with a lighter colour for each of us and bold font in the "article_evaluations" tab.

https://github.com/phytopathology/Reproducible.Plant.Pathology/commit/031b95131656e0e32ac93080c5744a26171420e9

I hope that makes sense. When I entered my information just now, I did find it easier to use with the colour shading differentiating the fields to check/enter and stuff that's there for analysis or auto-entered.

**Comment by adamhsparks** [comment]
Seeing some of Emerson's fields with "?" in them, regarding comments with multiple softwares used.

I've ranked the paper by the lowest rank. That is, if they used R and SAS, it got a 1 for availability because SAS is ranked as 1, no matter what R was ranked (would be 3).

I've used that in all cases where there are multiple values, e.g., R 3.4.3 was cited, but SAS was not given a version number, etc.

HTH.
